hooks/useClickOutside.ts

```diff
@@ -0,0 +1,33 @@
+import { useEffect } from 'react';
+import type { RefObject } from 'react';
+
+// eslint-disable-next-line no-unused-vars
+type TypeHandler = (event: MouseEvent | TouchEvent) => void;
```

Comment:
```suggestion
type TypeHandler = (_event: MouseEvent | TouchEvent) => void;
```

And we need to wait for @HinataKah0 to add the eslint changes.